### PR TITLE
Update BMC configuration

### PR DESCRIPTION
**Pull request by keedya**
- update obm setting can now be enabled in dynamic discovery graph

@RackHD/corecommitters @tannoa2 @uppalk1 @geoff-reid

### CODE REVIEW

File: docs/rackhd/configuration.rst [hunk 1]
```diff
@@ -241,172 +243,13 @@ These parameters beging with *HTTP* and *HTTPS*.
...
-Once the above steps are completed (edited and saved) the service needs to be restarted:
+A node gets discovered and the BMC IPMI comes up with a default username/password. User can automatically set
+IPMI OBM settings  using a  default user name('__rackhd__') and an auto generated password in rackHD by adding the following
+to RackHD ``config.json``:
 
 .. code-block:: shell
 
-    sudo service on-taskgraph start
-
+   "obmOption": true
```
Comment (yyscamper) [inline docs/rackhd/configuration.rst]: If you look at how the condition job is implemented, you will see the boolean type will not work well due to the boolean doesn't have a prototype function `toLowerCase`, you if you don't change the condition job code, at here, you need change `true` to `"true"`.

File: docs/rackhd/configuration.rst [hunk 2]
```diff
@@ -209,6 +209,8 @@ The following table describes the configuration parameters in config.json:
       - Enable trust proxy in express. Populate req.ip with left most IP address from the XForwardFor list.
 
         See documentation at https://expressjs.com/en/guide/behind-proxies.html
+    * - obmOption
+      - Allow rackHD to setup IPMI OBM settings on active dicovery by creating a new BMC user on the compute node.
```
Comment (yyscamper) [inline docs/rackhd/configuration.rst]: Please add some more words to describe the type of obmOption, as due to our implementation in condition job, the type of obmOption is very tricky, it should be `a boolean string`.

**Comment by yyscamper** [comment]
:+1: after fix above comment.

**Comment by keedya** [comment]
@yyscamper I tested with true as a booleen and it works.

**Comment by keedya** [comment]
@yyscamper comments addressed :)

**Comment by yyscamper** [comment]
@keedya : Thanks, I just realized the task option rendering "{{options.autoCreateObm}}" will convert it from boolean to string.

**Pull request merged**